web/src/components/apps/AppVersionHistory.jsx

```diff
@@ -349,8 +349,13 @@ class AppVersionHistory extends Component {
           {diffSummary.filesChanged > 0 ?
             <div className="DiffSummary u-marginRight--10">
               <span className="files">{diffSummary.filesChanged} files changed </span>
-              {!downstream.gitops?.enabled &&
-                <span className="u-fontSize--small replicated-link u-marginLeft--5" onClick={() => this.setState({ showDiffOverlay: true, firstSequence: previousSequence, secondSequence: version.parentSequence})}>View diff</span>
+              {!this.props.isHelmManaged && !downstream.gitops?.enabled &&
+                <span
+                  className="u-fontSize--small replicated-link u-marginLeft--5"
+                  onClick={() =>
+                    this.setState({ showDiffOverlay: true, firstSequence: previousSequence, secondSequence: version.parentSequence})}>
+                    View diff
+                </span>
```

Comment:
Is the intention to prevent all diffing of versions or just not showing the inline diff count and link? I noticed we still are showing the link to "Diff versions" on the top right. Just want to make sure that we hide that link depending on what the intention here is.

Reply:
The diff isn't very meaningful with Helm charts, so we're completely excluding the diff. I agree that the button up top should go.